packages/core/schematics/migrations.json

```
@@ -9,6 +9,11 @@
       "version": "19.0.0",
       "description": "Updates ExperimentalPendingTasks to PendingTasks",
       "factory": "./bundles/pending-tasks#migrate"
+    },
+    "http-xhr-backend": {
+      "version": "19.0.0",
+      "description": "Updates provideHttpClient to add withXhr when the XhrBackend was used",
```

NIT: Adding more information and some formatting like quotes would be great. 
Maybe a link to help users understand the change or at least a link to the API. 

```suggestion
      "description": "Adds 'withXhr' to 'provideHttpClient' function calls when 'XhrBackend' is used. For more information see: https://angular.dev/api/common/http/withXhr",
```